FeatureFlags.md

```diff
@@ -0,0 +1,25 @@
+The implementations of `amplify-codegen` plugin and the following code generators:
+- `appsync-modelgen-plugin` - generates model files to be used with Amplify Datastore. Accessed via CLI command `amplify codegen models`
+- `graphql-docs-generator` - generates GraphQL operations from the given GraphQL schema. Accessed via CLI command `amplify codegen statements` and `amplify-codegen`
+- `graphql-types-generator` - generates type definitions in given target language for the types defined in the GraphQL schema. Accessed via CLI command `amplify codegen types` and `amplify-codegen`
+
+are transferred to this repository from the [Amplify CLI repository](https://github.com/aws-amplify/amplify-cli/tree/master/packages). 
+
+In order to shield the existing customers from any complications as a result of this transfer, we provide the following Feature Flags that enable the customers to switch to the migrated package implementations at their own pace before the deprecation date:
+
+1. **useAppsyncModelgenPlugin**: When set to `true`, the `appsync-modelgen-plugin` implementation in this repository will be used for models generation. When set to `false`, the `amplify-codegen-appsync-model-plugin` implementation from [CLI repo](https://github.com/aws-amplify/amplify-cli/tree/master/packages/amplify-codegen-appsync-model-plugin) will be used for models generation. 
```

Comment:
I find in my `amplify/cli.json` that they are using all lowercases instead of camelCases. Can you double check on your side?
```JSON
{
  "features": {
    "graphqltransformer": {
      "addmissingownerfields": true,
      "validatetypenamereservedwords": true,
      "useexperimentalpipelinedtransformer": false,
      "enableiterativegsiupdates": false,
      "secondarykeyasgsi": true
    },
    "frontend-ios": {
      "enablexcodeintegration": true
    },
    "auth": {
      "enablecaseinsensitivity": true
    },
    "codegen": {
      "useappsyncmodelgenplugin": true,
      "usedocsgeneratorplugin": true,
      "usetypesgeneratorplugin": true
    },
    "appsync": {
      "generategraphqlpermissions": true
    }
  }
}
```

Reply:
Yes, CLI seems to convert the flags to lower cases. But [their docs](https://docs.amplify.aws/cli/reference/feature-flags#enableCaseInsensitivity) still maintain the camel case, like how these flags were registered. I believe we should follow their docs convention to avoid confusion. I think irrespective of case, the user can easily figure out what we're referring to here.